compiler/one-cmds/one-quantize

```diff
@@ -360,6 +366,42 @@ def _quantize(args):
         # run circle-quantizer
         _utils._run(circle_quantizer_cmd, err_prefix="circle_quantizer", logfile=f)
 
+        # evaluate
+        if _utils._is_valid_attr(args, 'test_data'):
+            circle_eval_diff_path = os.path.join(dir_path, 'circle-eval-diff')
+            tmp_fake_quant_model = os.path.join(
+                tmpdir,
+                os.path.splitext(os.path.basename(args.input_path))[0]) + '.fake_quant.circle'
+
+            # do fake quantization
+            fake_quantize_cmd = [circle_quantizer_path]
+            if _utils._is_valid_attr(args, 'verbose'):
+                fake_quantize_cmd.append('--verbose')
+            fake_quantize_cmd.append('--fake_quantize')
+            fake_quantize_cmd.append(getattr(args, 'output_path'))
+            fake_quantize_cmd.append(tmp_fake_quant_model)
+
+            f.write((' '.join(fake_quantize_cmd) + '\n').encode())
+            _utils._run(fake_quantize_cmd, err_prefix="circle_quantizer", logfile=f)
+
+            # compare fake-quant model and fp32 model
+            circle_eval_diff_cmd = [circle_eval_diff_path]
+            circle_eval_diff_cmd.append('--first_model')
+            circle_eval_diff_cmd.append(getattr(args, 'input_path'))
+            circle_eval_diff_cmd.append('--second_model')
+            circle_eval_diff_cmd.append(tmp_fake_quant_model)
+            circle_eval_diff_cmd.append('--first_input_data')
+            circle_eval_diff_cmd.append(getattr(args, 'test_data'))
+            circle_eval_diff_cmd.append('--second_input_data')
+            circle_eval_diff_cmd.append(getattr(args, 'test_data'))
+            if _utils._is_valid_attr(args, 'input_data_format'):
+                circle_eval_diff_cmd.append('--input_data_format')
+                circle_eval_diff_cmd.append(getattr(args, 'input_data_format'))
+            circle_eval_diff_cmd.append('--print_mae')
+            circle_eval_diff_cmd.append('--print_mpeir')
+
+            f.write((' '.join(circle_eval_diff_cmd) + '\n').encode())
+            _utils._run(circle_eval_diff_cmd, err_prefix="circle_eval_diff", logfile=f)
```

Comment:
Seems `quantize` method will become longer and longer. How about make them several functions later?

Reply:
Above two comments are optional.

Reply:
> Also, adding test about this change would be helpful.

I've added tests.